osidb/middleware.py

```diff
@@ -32,3 +34,41 @@ def __call__(self, request):
             _logger.info(f"ACLs set from middleware for user {request.user.username}")
             set_user_acls(request.user.groups.all())
         return self.get_response(request)
+
+
+def OsidbHistoryMiddleware(get_response):
```

Comment:
Is there a reason why we're defining this middleware as a function while the other is a class? I'm okay with either option but if there's no strong reason to choose one over the other I'd rather keep it consistent at least.